content/developer/embedding/index.md

```diff
@@ -7,7 +7,6 @@ category: help
 menu:
   developer:
     identifier: embedding
-    weight: 100
 aliases:
 - /features/embedding
```

Comment:
Incorrect Alias

Reply:
It seems this file is actually in 2 places. I'm going to remove the one in the help section and create an alias to it in the one on the developer side.

Reply:
I've implemented this feedback